<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #363636</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>162</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 28px</string>
      </property>
      <property name="text">
       <string>Goal</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addGoalButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>Add Goal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeGoalButton">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>420</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>Remove Goal</string>
   </property>
  </widget>
  
  <widget class="QTextEdit" name="goalInputBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color:black;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="goalList_Widget">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>111</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editGoalButton">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>420</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
 <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; color: black;</string>
   </property>
   <property name="text">
    <string>Import Goal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; color: black;</string>
   </property>
   <property name="text">
    <string>Export Goal</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>